### PR TITLE
Add Hyper-V svirt support

### PR DESCRIPTION
**Pull request by mnowaksuse**
We need ssh -X connection so we can run Xvnc via vncserver.

### CODE REVIEW

**Comment by mnowaksuse** [comment]
Required: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1348

**Comment by coolo** [comment]
call sshCommand with , 1 instead of hardcoding -X

**Comment by okurz** [comment]
+1

**Comment by mnowaksuse** [comment]
It seems that I don't need `-X` at all.

File: cpanfile [hunk 1]
```diff
@@ -33,6 +33,7 @@ requires 'Mojolicious::Lite';
 requires 'Net::DBus';
 requires 'Net::SNMP';
 requires 'Net::SSH2';
+requires 'Net::Telnet';
```
Comment (coolo) [inline cpanfile]: this seems to be a bit undermotivated

Comment (coolo) [inline cpanfile]: Cherry pick https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/3880f711b9dc1e73ecf77e171d727d3713246d49 and revert that line here

**Comment by okurz** [comment]
LGTM

**Pull request merged**